application/engine/services/autopilot_daemon.py

```
@@ -707,22 +753,382 @@ async def _handle_writing(self, novel: Novel):
+    ) -> tuple[bool, str]:
+        normalized_outline = self._normalize_seam_text(outline)
+        anchor_score = self._calculate_outline_anchor_score(normalized_outline, seam)
+        reset_markers = (
+            "次日", "翌日", "第二天", "天亮", "清晨", "黎明", "晨光", "早晨",
+        )
+        has_reset_marker = any(marker in outline for marker in reset_markers)
+
+        if anchor_score >= 2:
+            return False, ""
+        if has_reset_marker and anchor_score == 0:
+            return True, "下一章大纲另起时间场景，且未承接上一章接缝"
```

Ruff (0.15.10)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)